feature-about/src/test/java/io/github/droidkaigi/confsched2022/feature/about/AboutScreenRobot.kt

```diff
@@ -1,13 +1,19 @@
 package io.github.droidkaigi.confsched2022.feature.about
 
 import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.onNodeWithText
 import io.github.droidkaigi.confsched2022.testing.RobotTestRule
 import javax.inject.Inject
 
 class AboutScreenRobot @Inject constructor() {
 
     context(RobotTestRule)
     fun checkAboutVisible() {
+        composeTestRule.onNodeWithText("What is DroidKaigi?")
+        composeTestRule.onNodeWithText(
+            "DroidKaigiはエンジニアが主役のAndroidカンファレンスです。" +
+                "\\nAndroid技術情報の共有とコミュニケーションを目的に、2022年10月5日(水)〜7日(金)の3日間開催します。"
+        )
```

Comment:
I don't think we need to check all the text. 🙏 
It can be a change-detector test. 
https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html

```suggestion
```